### PR TITLE
Fix broken link to auth getting started page

### PR DESCRIPTION
**Pull request by timngyn**
_Issue #, if available:_ #4952

_Description of changes:_
- Fix broken link

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/start/getting-started/react/auth.mdx [hunk 1]
```diff
@@ -226,6 +226,6 @@ In this example, you used the Amplify React UI library and the `withAuthenticato
 
 You can also [customize this component](https://ui.docs.amplify.aws/react/components/authenticator/customization) to add or remove fields, update styling, or other configurations. You can even [override function calls](https://ui.docs.amplify.aws/react/components/authenticator/customization#override-function-calls) if needed.
 
-In addition to the `withAuthenticator`, you can build [custom authentication flows Amplify Libraries for JS](lib/auth/getting-started/q/platform/js/). Amplify's `Auth` class has over 30 methods including `signUp`, `signIn`, `forgotPassword`, and `signOut` that allow you full control over all aspects of the user authentication flow. Check out the complete API [here](https://aws-amplify.github.io/amplify-js/api/classes/authclass.html).
+In addition to the `withAuthenticator`, you can build [custom authentication flows Amplify Libraries for JS](/lib/auth/getting-started). Amplify's `Auth` class has over 30 methods including `signUp`, `signIn`, `forgotPassword`, and `signOut` that allow you full control over all aspects of the user authentication flow. Check out the complete API [here](https://aws-amplify.github.io/amplify-js/api/classes/authclass.html).
```
Comment (josefaidt) [inline src/fragments/start/getting-started/react/auth.mdx:229]: I think deep-linking to Getting Started for JavaScript makes sense given the link text "...Amplify Libraries for JS", but the preceding slash LGTM

Comment (timngyn) [reply src/fragments/start/getting-started/react/auth.mdx:229]: Doesn't that break the link though? I did initially try it out locally with the `/q/platform/js` but once I clicked the link, it added an extra `/q/platform/js`

Comment (josefaidt) [reply src/fragments/start/getting-started/react/auth.mdx:229]: oh interesting! and I see it works fine on the staging site

**Review by josefaidt — APPROVED** [review]
LGTM 🚢

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**